mod/ckeditor/views/default/elgg/ckeditor/set-basepath.js

```diff
@@ -1,2 +1,7 @@
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+/**
+ * This view extends elgg.js and sets ckeditor basepath before jquery.ckeditor is loaded
+ */
+require(['elgg'], function(elgg) {
+	// This global variable must be set before the editor script loading.
+	CKEDITOR_BASEPATH = elgg.get_simplecache_url('ckeditor/');
```

Comment:
`get_simplecache_url` asks for a view name. If we're going to support an arbitrary path to be passed in we should document that in the JSdoc and maybe make a test for it.